tests/parm/model_configure_rrfs_conus13km.IN

```diff
@@ -13,7 +12,6 @@ restart_interval:        @[RESTART_INTERVAL]
 output_1st_tstep_rst:    .false.
 
 memuse_verbose:          .false.
-print_esmf:              .true.
 RUN_CONTINUE:            .false.
 ENS_SPS:                 .false.
```

Comment:
Not strictly related to the changes in this PR but `RUN_CONTINUE` and `ENS_SPS` could be removed as well.